webcompat/static/css/src/issue.scss

```diff
@@ -21,7 +21,7 @@
 }
 
 .issue-labels {
-  max-width: calc(100% - var(--unit-space));
+  max-width: calc(100% - #{$unit-space});
```

Comment:
(I've never used sass, what does the `#{}` mean here?)

Reply:
http://sass-lang.com/documentation/file.SASS_REFERENCE.html#interpolation_

It allows to interpolate variables in combined statements. It's like ``${someVar}`` in JavaScript template strings using backticks.

Reply:
Ah, cool. Thanks!